pkg/core/config/main.go

```diff
@@ -56,8 +56,10 @@ type Spec struct {
 	AutoDiscovery autodiscovery.Config `yaml:",omitempty"`
 	// Title is used for the full pipeline
 	Title string `yaml:",omitempty"`
-	// PullRequests defines the list of Pull Request configuration which need to be managed
-	PullRequests map[string]pullrequest.Config `yaml:",omitempty"`
+	// !Deprecated in favor of `actions`
+	PullRequests map[string]action.Config `yaml:",omitempty"`
```

Comment:
```suggestion
	PullRequests map[string]action.Config `yaml:",omitempty" jsonschema:"-"`
```

Adding the tag jsonschema:"-" means that `pullrequests` will trigger a warning in IDE using jsonschema